apps/openmw/mwgui/savegamedialog.cpp

```diff
@@ -361,6 +378,11 @@ namespace MWGui
             << MWBase::Environment::get().getWorld()->getMonthName(mCurrentSlot->mProfile.mInGameTime.mMonth)
             <<  " " << hour << " " << (pm ? "#{sSaveMenuHelp05}" : "#{sSaveMenuHelp04}");
 
+        if (Settings::Manager::getBool("timeplayed","Saves"))
+        {
+            text << "\n" << "Time played: " << formatTimeplayed((int)floor(mCurrentSlot->mProfile.mTimePlayed));
```

Comment:
Maybe we should change mTimePlayed to be in long int and not double since we don't need milliseconds.